flink-connectors/flink-sql-connector-hbase-2.2/src/main/resources/hbase-default.xml

```diff
@@ -0,0 +1,1558 @@
+			which will tail the logs and replicate the mutatations to region replicas for tables that
+			have region replication > 1. If this is enabled once, disabling this replication also
+			requires disabling the replication peer using shell or ReplicationAdmin java class.
+			Replication to secondary region replicas works over standard inter-cluster replication.
+			So replication, if disabled explicitly, also has to be enabled by setting "hbase.replication"
+			to true for this feature to work.
+		</description>
+	</property>
+	<property>
+		<name>hbase.security.visibility.mutations.checkauths</name>
+		<value>false</value>
+		<description>
+			This property if enabled, will check whether the labels in the visibility expression are associated
+			with the user issuing the mutation
+		</description>
+	</property>
+	<property>
+		<name>hbase.http.max.threads</name>
+		<value>10</value>
+		<description>
+			The maximum number of threads that the HTTP Server will create in its
+			ThreadPool.
+		</description>
+	</property>
+	<property>
+		<name>hbase.replication.source.maxthreads</name>
+		<value>10</value>
+		<description>
+			The maximum number of threads any replication source will use for
+			shipping edits to the sinks in parallel. This also limits the number of
+			chunks each replication batch is broken into.
+			Larger values can improve the replication throughput between the master and
+			slave clusters. The default of 10 will rarely need to be changed.
+		</description>
+	</property>
+	<!-- Static Web User Filter properties. -->
+	<property>
+		<description>
+			The user name to filter as, on static web filters
+			while rendering content. An example use is the HDFS
+			web UI (user to be used for browsing files).
+		</description>
+		<name>hbase.http.staticuser.user</name>
+		<value>dr.stack</value>
+	</property>
+	<property>
+		<name>hbase.regionserver.handler.abort.on.error.percent</name>
+		<value>0.5</value>
+		<description>The percent of region server RPC threads failed to abort RS.
+			-1 Disable aborting; 0 Abort if even a single handler has died;
+			0.x Abort only when this percent of handlers have died;
+			1 Abort only all of the handers have died.</description>
+	</property>
+	<property>
+		<name>hbase.snapshot.master.timeout.millis</name>
+		<value>300000</value>
+		<description>
+			Timeout for master for the snapshot procedure execution
+		</description>
+	</property>
+	<property>
+		<name>hbase.snapshot.region.timeout</name>
+		<value>300000</value>
+		<description>
+			Timeout for regionservers to keep threads in snapshot request pool waiting
+		</description>
+	</property>
+	<property>
+		<name>hbase.rpc.rows.warning.threshold</name>
+		<value>5000</value>
+		<description>
+			Number of rows in a batch operation above which a warning will be logged.
+		</description>
+	</property>
+
+	<!--NOTE: HBase client try to load the class that configured in hbase-default.xml. -->
```

Comment:
we'd better reference the hbase-default.xml from `~/.m2/repository/org/apache/hbase/hbase-common/2.2.3/hbase-common-2.2.3.jar!/hbase-default.xml` and replace all hbase class with our flink shaded class.

Reply:
I'm not sure what you mean here. All the classes are already the flink shaded classes. I also don't understand why would it be good to reference hbase-default.xml from the local mmaven repository. Please clarify.

Reply:
The hbase client will load the class from non-shaded hbase class from `hbase-default.xml`, we found the problem when we make the hbase 1 sql connector.  ^_^

see more here[1] https://issues.apache.org/jira/browse/FLINK-14359?focusedCommentId=16949271&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16949271

Reply:
I still don't see the problem here, the comment you were referring to states that we should use the shaded classes, and this is what is happening here, hbase-default.xml filres both in 1.4 and 2.2 are using the shaded classes.

Reply:
Yes, here we have used shaded classes, but `hbase 2.2` and `hbase 1.4` have different `hbase-default.xml` and the shaded classes are also different(`hbase1` has 17 classes need to shade, but `hbase2` has 19). So I suggest we use corresponding version of `hbase-default.xml` and shade all classes.

Reply:
Ah, ok I see now what you mean. I've added the proper `hbase-default.xml`.

As I see in the old connector hbase-server is excluded from shading, yet a lot of classes that are supposed to come from that are referenced in `hbase-default.xml`, for example `org.apache.flink.hbase.shaded.org.apache.hadoop.hbase.master.cleaner.TimeToLiveLogCleaner`. Isn't that a problem?

Reply:
I'm not sure it's a problem or not.
The shaded classes in `hbase-default.xml` is mainly used by `hbase-client` in flink hbase connector, all `hbase-server `related class should not be used here. 
 But I think we can only replace client-related class with shaded class as your concern, this should be an improvement.

Reply:
I updated the pom.xml/hbase-default.xml/NOTICE file for hbase2 sql module https://github.com/leonardBang/flink/commit/ed9e19f60c6b8a8f0ceb2279b01b97e24a755a4d, I think we need update them, hope this can help you a little.

Reply:
I have no other concerns, thanks for the great contribution.